style.css

```diff
@@ -53,34 +116,98 @@ legend {
 button {
   background-color: white !important;
   height: 30px;
-  margin-bottom: 20px;
+  margin-bottom: 24px;
   position: relative;
   bottom: 6px;
   box-shadow: 5px 5px black;
   border: 4px solid black;
-}
-
-.underline {
-  width: 240px;
-  margin-top: 40px;
-  margin-bottom: 40px;
+  margin-left: 8%;
 }
 
 #add-new {
-  width: 390px;
+  width: 100%;
   display: flex;
   flex-direction: column;
-  margin-left: -6%;
   align-items: center;
+  justify-content: center;
 }
 
 .form {
   border: 4px solid black;
-  margin-left: 90px;
+  margin-left: 5px;
+}
+
+footer {
+  width: 97%;
+  height: 3rem;
+  border: 3px solid #000;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  bottom: 12px;
+  padding-left: 1rem;
+  margin: auto;
+}
+
+.time {
+  left: 83%;
+  font-size: 13px;
+  position: relative;
+  font-weight: 700;
+  width: 17%;
+}
+
+.contact-us {
+  display: flex;
+}
```

Comment:
- [ ] The `contact-us` section is overflowing from the view window. Kindly make changes to your styling. :pray: 

![Screenshot from 2021-09-23 22-10-20](https://user-images.githubusercontent.com/64894927/134544557-8943a05c-b7a4-4e6b-b715-d66f186a58cc.png)

Reply:
fixedd